src/installer/corehost/cli/json_parser.cpp

```diff
@@ -74,7 +74,7 @@ void json_parser_t::realloc_buffer(size_t size)
     m_json[size] = '\0';
 }
 
-bool json_parser_t::parse_json(char* data, int64_t size, const pal::string_t& context)
+bool json_parser_t::parse_raw_data(char* data, int64_t size, const pal::string_t& context)
 {
     constexpr auto flags = rapidjson::ParseFlag::kParseStopWhenDoneFlag | rapidjson::ParseFlag::kParseCommentsFlag;
```

Comment:
Please assert that `data` isn't `nullptr` here. Unless `m_document` is our code.